src/Data/Swagger/Declare.hs

```diff
@@ -151,12 +154,11 @@ instance MonadDeclare d m => MonadDeclare d (ContT r m) where
   declare = lift . declare
   look = lift look
 
--- Note: deprecated, fails with -Wall
-instance (Error e, MonadDeclare d m) => MonadDeclare d (ErrorT e m) where
+#if __GLASGOW_HASKELL__ >= 710
```

Comment:
We should depend on `transformers-compat >= 0.3` (if we don't), then `ExceptT` will always be there.